JavaScript/Algorithms/vowelCount.js

```diff
@@ -0,0 +1,22 @@
+//Program to count the number of vowels in a sentence
+const vowels=["a","e","i","o","u","A","E","I","O","U"]
+function vowelCount(str){
+    let count=0
+    for(let character of str){
+        if(vowels.includes(character)){
+            count++;
+        }
+    }
+    return count;
+}
+prompt("Enter string:")
+const count=vowelCount(str)
+console.log(count)
```

Comment:
This part is buggy. You are not even assigning `str`

Reply:
I have corrected the code